docs/coding-guidelines/javascript.md

```diff
@@ -68,50 +71,45 @@ foo( {
 } );
 
 foo( data, function() {
-    // Do stuff
+	// Do stuff
 } );
 
 foo( function() {
-    // Do stuff
+	// Do stuff
 }.bind( this ) );
 
 foo( function() {
-    // Do stuff
+	// Do stuff
 }, options );
-
 ```
 
 ## Examples of Good Spacing
 
 ```js
-var i;
-
 if ( condition ) {
-    doSomething( 'with a string' );
+	doSomething( 'with a string' );
 } else if ( otherCondition ) {
-    otherThing( {
-        key: value,
-        otherKey: otherValue
-    } );
+	otherThing( {
+		key: value,
+		otherKey: otherValue
+	} );
 } else {
-    somethingElse( true );
+	somethingElse( true );
 }
 
-// Unlike jQuery, WordPress prefers a space after the ! negation operator.
-// This is also done to conform to our PHP standards.
 while ( ! condition ) {
-    iterating++;
+	iterating++;
 }
 
-for ( i = 0; i < 100; i++ ) {
-    object[ array[ i ] ] = someFn( i );
-    $( '.container' ).val( array[ i ] );
+for ( let i = 0; i < 100; i++ ) {
+	object[ array[ i ] ] = someFn( i );
+	$( '.container' ).val( array[ i ] );
```

Comment:
>oh my. I hope to never find this actual code in a review

Removed this specific line in 4d171dd3b56e8567f60d3258274dd797520c0d65 . There's a point to be made that this could be a `reduce` or `forEach`, but I think the point is to demonstrate `for` spacing.